src/ui/FileViewer/index.scss

```diff
@@ -106,3 +107,25 @@
     }
   }
 }
+
+.sendbird-file-viewer-arrow--left,
+.sendbird-file-viewer-arrow--right {
+  cursor: pointer;
+  position: absolute;
+  // half-joke, half-serious, its better to have some color change on hover
+  // ask gabie for suggestion
+  &:hover {
+    border-radius: 50%;
+    background-color: rgb(98, 144, 144);
+    box-shadow: 0 0 0 2px #fff, 0 0 0 4px rgb(98, 144, 144);
+  }
```

Comment:
is good UX, ask gabie for styling suggestion

Reply:
I asked she said no slide button hover effect